ocs_ci/utility/utils.py

```diff
@@ -907,6 +913,30 @@ def get_cluster_version_info():
     return cluster_version_info
 
 
+def get_ocs_build_number():
+    """
+    Gets the build number for ocs operator
+
+    Return:
+        str: build number for ocs operator version
+
+    """
+    from ocs_ci.ocs.resources.catalog_source import CatalogSource
```

Comment:
Do we really need to do imports here? I prefer to do import at the top of the module.

Reply:
There was cyclic dependency problem if I add these imports at the top and run was failing, hence made it local import

Reply:
Ok then, but for the second import from constants I guess there is not cyclic dependency so at least this should be moved at the top

Reply:
Done